spec/index.bs

```diff
@@ -2,7 +2,8 @@
 Title: Contact Picker API
 Shortname: contact-picker
 Level: 1
-Status: UD
+Status: FPWD
+TR: https://w3c.github.io/contact-api/spec
```

Comment:
I didn't find a draft of the spec under https://www.w3.org/TR/, so I linked to the github spec for now

Reply:
thanks a lot, @rayankans !

The "TR" meta-data is usually decided by our Sys-Team, so please hold this Pull Request until we publish the FPWD next Tuesday :)

Reply:
Oh, sorry, the publication is waiting on a Former Objection to the new charter...

See [FPWD Request for Contacts Picker API #457](https://github.com/w3c/transitions/issues/457)